libs/ui-toolkit/src/components/tabs/tabs.tsx

```diff
@@ -122,6 +140,23 @@ export const Tabs = ({
           );
         })}
       </div>
+      {settingsOpened && !!settings ? (
```

Comment:
For me this UI doesn't look tidy. I don't like the pane overlaying the tabs bar and how the button has to dropdown.

Reply:
It needs a left border too.

Reply:
Pr to merge here: https://github.com/vegaprotocol/frontend-monorepo/pull/5532

Reply:
Resolving this as we are taking a different approach